lib/bambora/client.rb

```diff
@@ -1,14 +1,16 @@
 # frozen_string_literal: true
 
+require 'base64'
+require 'excon'
```

Comment:
These are required explicitly in order to be used in the client.

Reply:
I'm curious about this. When I tested the client myself, it worked without these lines. I may have run `rake build` when testing.

Reply:
Yes it also seemed to work for me before, but I can't remember how I got it to work. `base64` is not part of the core library, and there is a call to `Base64` at the bottom of this file.